packages/test/test-service-load/package.json

```diff
@@ -28,6 +28,7 @@
     "lint:fix": "npm run eslint:fix",
     "start": "node ./dist/nodeStressTest.js",
     "start:mini": "node ./dist/nodeStressTest.js --profile mini",
+    "start:hi": "node ./dist/mini.js",
     "tsc": "tsc",
```

Comment:
i think you need to add mocha dev dependecies here

Reply:
"@types/mocha": "^8.2.2", this one is already in dev dependency. is there another one?